include/rocksdb/utilities/object_registry.h

```diff
@@ -278,6 +278,9 @@ class ObjectRegistry {
   static std::shared_ptr<ObjectRegistry> Default();
   explicit ObjectRegistry(const std::shared_ptr<ObjectRegistry>& parent)
       : parent_(parent) {}
+  explicit ObjectRegistry(const std::shared_ptr<ObjectLibrary>& library) {
```

Comment:
Nit: when the point of the function/ctor is to save a copy of a shared_ptr, I think it's better to pass by shared_ptr value